src/commands/codepush/deployment/lib/date-helper.ts

```diff
@@ -0,0 +1,13 @@
+import * as moment from "moment";
+
+export function formatDate(unixOffset: number): string {
```

Comment:
I have synced with the portal date retrieval code. Yeah, the current date format is unix timestamps. We tried to keep the same date format as the original code-push cli.